app/models/relationship.rb

```diff
@@ -0,0 +1,6 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
```

Comment:
```suggestion
  belongs_to :follower, class_name: User.name
```

This has the benefit that if you ever change the class name `User` to something like `MicropostAppUser` then you will get a "compile-time" error and not a "runtime" error.  OK, it will still be runtime, but it will happen further up the stack.